fudge-ratio/java/servlets/FudgeRatio.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+import com.google.cloud.translate.Translate;
```

Comment:
Hello, just my opinion:
Do we really need all does imports?

Reply:
My bad, I had copied the imports from my portfolio servlet and forgot to go through and delete the unnecessary ones.